libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/OAuth2.java

```diff
@@ -233,58 +226,6 @@ public static URI getAuthorizationUrl(URI loginServer, String clientId, String c
         return URI.create(sb.toString());
     }
 
-    /**
-     * Returns an IDP 'frontdoor' URL configured with the SP's configuration.
-     *
-     * @param instanceUrl IDP's instance URL.
-     * @param accessToken IDP's access token.
-     * @param loginUrl SP's login URL.
-     * @param displayType IDP's display type.
-     * @param clientId SP's client ID.
-     * @param callbackUrl SP's callback URL.
-     * @param scopes SP's scopes.
-     * @param codeChallenge SP's code challenge.
-     * @return A 'frontdoor' URL that the IDP can load in a WebView.
-     */
-    public static URI getIDPFrontdoorUrl(String instanceUrl, String accessToken, String loginUrl,
```

Comment:
getIDPFrontdoorUrl = getIDPApprovalUrl through front door
and getIDPApprovalUrl = getWebServerApprovalUrl which now can be obtained by calling getAuthorizationUrl